### DISCUSSION TITLE
Azure - Volume Affinity Issues When Restoring A Backup From One Region To Another

### DISCUSSION THREAD

**Discussion by HizzyCO** [vmware-tanzu/velero#3262] [Community support Q&A] [answered]
I am following the documentation on restoring a back up to a new cluster but when trying to restore the volumes there is an affinity issue. The secondary cluster is located in a different region than the original cluster and its snapshots. This is the node affinity for the volume and the issue is the the match expression is the old region, the new one is southcentralus. For DR the new cluster will always be in a different region. Does Velero support this?

Velero Version: 4.1.0
Kube Version: 1.16.13
AKS Environment

Volume Affinity:
`
nodeAffinity:
    required:
      nodeSelectorTerms:
      - matchExpressions:
        - key: failure-domain.beta.kubernetes.io/region
          operator: In
          values:
          - northcentralus
`

**Comment by dsu-igeek** [comment, marked as answer]
For cross-AZ movement you will need to use Restic and backup the volume data to the Azure object store.  Some info on migration is here: https://velero.io/docs/v1.5/migration-case/  Some info on Restic is here: https://velero.io/docs/v1.5/restic/

**Reply by HizzyCO** [reply]
Thank you I will dive in and confirm

**Reply by HizzyCO** [reply]
quick question I was reading over restic limitations how can I detemine this?

For this reason, restic integration can only backup volumes that are mounted by a pod and not directly from the PVC.

pod
`
Volumes:
  datadir:
    Type:       PersistentVolumeClaim (a reference to a PersistentVolumeClaim in the same namespace)
`
pvc
`
VolumeMode:    Filesystem
Status:                Bound
Annotations:      pv.kubernetes.io/bind-completed: yes
                          pv.kubernetes.io/bound-by-controller: yes
                          volume.beta.kubernetes.io/storage-provisioner: kubernetes.io/azure-disk
`
pv 
`
Source:
    Type:         AzureDisk (an Azure Data Disk mount on the host and bind mount to the pod)
`
